commands/pass.go

```diff
@@ -0,0 +1,59 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/bndw/pick/utils/pswdgen"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "pass",
+		Short: "Generate a password without storing it",
+		Long:  "The pass command is used to generate a password without storing it.",
+		Run: func(cmd *cobra.Command, args []string) {
+			runCommand(Pass, cmd, args)
+		},
+	}
+	cmd.Flags().Int("length", pswdgen.DefaultPasswordLength, "the length of the generated password")
```

Comment:
Here's what I expected:

1. the `add` command respects mode set in the config
2. the `pass` command defaults to "non-interactive", regardless of what is in the config.

Additionally, I suggested a `--mode` flag to the `pass` command to allow overriding of #2.